bundles/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/handler/KM200GatewayHandler.java

```diff
@@ -109,6 +112,18 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
+        try {
+            int maxKeyLen = Cipher.getMaxAllowedKeyLength("AES/ECB/NoPadding");
+            if (maxKeyLen <= 128) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "Java Cryptography Extension (JCE) have to be installed");
+                logger.warn("Java Cryptography Extension (JCE) have to be installed");
+                return;
+            }
+        } catch (NoSuchAlgorithmException e) {
+            logger.warn("AES encoding not supported", e);
```

Comment:
Please replace the call to `logger.warn` by a call to `updateStatus` to avoid letting your thing in INITIALIZIUNG state.

Reply:
Ok, done